projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```diff
@@ -118,3 +118,13 @@
 		}
 	}
 }
+
+.wp-block-jetpack-subscriptions__sb_post_end {
```

Comment:
Can we wrap in `group` or other block instead to add these?

Reply:
You mean `<!-- wp:group -->` or just a class `wp-block-group`? Not sure, never used them :)

Reply:
We'll need some design for final result, but otherwise best workflow is:

- In the block editor, create the visual you're aiming for (group block with center-align, and margin settings)
- Switch to code view in editor
- Copy paste the block code
- In PHP, `do_blocks( 'copy pasted blocks' )`

[Example](https://github.com/Automattic/jetpack/blob/f7dd7fdcc30ddb8a8b81201616b298d1430fdad0/projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php#L1063-L1083)

Reply:
What if I want to add some styling that is not supported in the Editor? Add it by the custom class?

Reply:
That's fine but I'd try avoid as much as possible to simplify things later down the road.

Blocks support custom classnames in the editor as well.

Reply:
Should be fixed already, thanks!